data/json/recipes/armor/storage.json

```diff
@@ -84,6 +84,21 @@
     "using": [ [ "strap_large", 1 ], [ "tailoring_cotton_patchwork", 2 ], [ "fastener_small", 2 ] ],
     "proficiencies": [ { "proficiency": "prof_closures" }, { "proficiency": "prof_closures_waterproofing" } ]
   },
+  {
+    "result": "leather_stone_pouch",
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "category": "CC_ARMOR",
+    "subcategory": "CSC_ARMOR_STORAGE",
+    "skill_used": "tailor",
+    "difficulty": 3,
+    "skills_required": [ [ "throw", 1 ] ],
```

Comment:
I think this is unneeded. I still should be able to craft this pouch even if I don't have any throwing experience at all.

Reply:
I think the original author intended that once the user is comfortable using stones/slings - they figure out the best way to place the pouch on their body... But np, I'll push a version without it.